templates/theme-footer.html

```diff
@@ -4,23 +4,19 @@
 <%! from datetime import datetime %>
 <%! from pytz import UTC %>
 
-<div class="wrapper-footer">
+<div class="wrapper-footer gray-gradient">
   <footer>
     <div class="colophon">
       <nav class="nav-colophon">
         <ol>
+          <li><a href="https://www.enthought.com/company/#about">About Enthought</a></li>
           <li><a href="${reverse('tos')}">Terms of Service</a></li>
-          <li><a href="${reverse('tos')}#privacy">Privacy Policy</a></li>
-          <li><a href="${reverse('tos')}#honor">Honor Code</a></li>
-          <li><a href="${reverse('tos')}#copyright">Copyright</a></li>
-          <li><a href="${reverse('about')}#careers">Careers</a></li>
-          <li><a href="${reverse('about')}#contact">Contact</a></li>
           <li><a href="${marketing_link('FAQ')}">Help</a></li>
```

Comment:
We can put back "Terms of Service" and "Help" by just directing them to corresponding URLs on the enthought website / ETOD website.